pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -2071,6 +2077,11 @@ def test_td64arr_div_numeric_array(self, box_with_array, vector, any_real_dtype)
             result = tdser / vector.astype(object)
             if box_with_array is pd.DataFrame:
                 expected = [tdser.iloc[0, n] / vector[n] for n in range(len(vector))]
+                if using_array_manager:
+                    # https://github.com/pandas-dev/pandas/issues/39750
+                    # third column with all-NaT as result doesn't get preserved
+                    # as timedelta64 dtype
+                    result[2] = pd.array(["NaT", "NaT"], dtype="timedelta64[ns]")
```

Comment:
my intuition is we should edit expected, not result.  is there a reason you did it this way?

Reply:
Just because there was already a `if box_with_array is pd.DataFrame` branch here, but can also add one below after the final `expected` is created and edit that one.

Reply:
updated

Reply:
thanks